.changeset/cute-taxis-double.md

```diff
@@ -0,0 +1,18 @@
+---
+'@backstage/backend-defaults': minor
+'@backstage/integration': minor
+---
+
+**BREAKING** Bitbucket cloud now supports API token as App password will be deprecated soon.
```

Comment:
```suggestion
**Attention:** Support for Bitbucket Cloud's API token was added. App password is deprecated (no new creation from September 9, 2025) and will be removed on June 9, 2026.
```

So far, it is still a minor change, not a major/breaking change due to the provided backwards compatibility.